Application Management/ExceptionHandlingSecondaryUIThread/SecondaryUIThreadWindow.xaml

```diff
@@ -5,7 +5,8 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:ExceptionHandlingSecondaryUIThread"
         mc:Ignorable="d"
-        Title="SecondaryUIThreadWindow" Height="300" Width="300">
+        Title="SecondaryUIThreadWindow" Height="300" Width="300"
+        Closed="SecondaryUIThreadWindow_OnClosed">
```

Comment:
Thanks, @caioproiete.  The addition of Closed="SecondaryUiThreadWindow_Closed" to the SecondaryUIThreadWindow.xaml's Window element line 9 looks good.  nit: Could you revert the formatting changes to SecondaryUIThreadWindow.cs?

Reply:
Thanks, @ryalanms, for reviewing.

The formatting was added automatically by Visual Studio 2017 upon saving the file - it seems by default it uses tabs instead of spaces. I fixed the formatting in the file and added an `.editorconfig` at the root folder so it doesn't happen again.